joint_limits:
  $(arg prefix)joint_1:
    has_position_limits: true
    min_position: -6.28
    max_position: 6.28
    has_velocity_limits: true
    max_velocity: 0.17
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 39.0

  $(arg prefix)joint_2:
    has_position_limits: true
    min_position: -2.41
    max_position: 2.41
    has_velocity_limits: true
    max_velocity: 0.17
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 39.0

  $(arg prefix)joint_3:
    has_position_limits: true
    min_position: -6.28
    max_position: 6.28
    has_velocity_limits: true
    max_velocity: 0.17
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 39.0

  $(arg prefix)joint_4:
    has_position_limits: true
    min_position: -2.66
    max_position: 2.66
    has_velocity_limits: true
    max_velocity: 0.17
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 39.0

  $(arg prefix)joint_5:
    has_position_limits: true
    min_position: -6.28
    max_position: 6.28
    has_velocity_limits: true
    max_velocity: 0.17
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 9.0

  $(arg prefix)joint_6:
    has_position_limits: true
    min_position: -2.23
    max_position: 2.23
    has_velocity_limits: true
    max_velocity: 0.17
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 9.0

  $(arg prefix)joint_7:
    has_position_limits: true
    min_position: -6.28
    max_position: 6.28
    has_velocity_limits: true
    max_velocity: 0.17
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 9.0
